### PR TITLE
Changed the way we get error not found message to its own slice zone

### PR DESCRIPTION
**Pull request by Churchoo**
Added a new Slice Zone and removed the error message from the old Slice Zone

### CODE REVIEW

**Comment by Churchoo** [comment]
Need to try fix enslint

**Comment by Churchoo** [comment]
I copied the package.json and package-lock.json from main to ensure that the checks passes since I believe axios and other things changed will caused checks to fail

**Comment by Churchoo** [comment]
you can view the ticket here https://linear.app/bankgreen/issue/GROOT-219/sfi-page-add-error-not-found-message-to-prismic-ecobankspage
it adds an error message if you no banks are shown when using filters on ecobankspage

**Comment by AlBillington** [comment]
I reverted the package and package-lock changes, seems to be good now

File: slices/ErrorMessage/index.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  ]),
+)
+</script>
+
+<template>
+  <section
+    :data-slice-type="slice.slice_type"
+    :data-slice-variation="slice.variation"
+  >
+    {{ slice.primary.title }}
```
Comment (AlBillington) [inline slices/ErrorMessage/index.vue:21]: should this be larger like h3 maybe?

**Review by AlBillington — APPROVED** [review]

**Comment by AlBillington** [comment]
tested this locally and LGTM, the title is the same size as the description which looks odd but non blocking

File: slices/ErrorMessage/index.vue (hunk 1 above)
Comment (Churchoo) [reply slices/ErrorMessage/index.vue:21]: sure that sounds like a good idea

**Pull request merged**